Chummer/data/gear.xml

```diff
@@ -1532,6 +1532,68 @@
       </weaponbonus>
       <ammoforweapontype>firefighting cannons</ammoforweapontype>
     </gear>
+	<gear>
+      <id>d71ea9c1-9b5f-428e-b849-7d665eb31c5e</id>
+      <name>Ammo: Sling-Shot Acid Capsules</name>
```

Comment:
Be consistent with the spelling of "slingshot"/"sling-shot". Either it has a hyphen everywhere or it is one word everywhere, don't mix up the two. This applies to all of the other slingshot ammo you added, though I will only make this comment for this entry.

Reply:
This one is a bit odd.  The capsule rounds are technically only listed for the Ares Giantslayer, which is printed as "slingshot" but the other 3 are for the Ontario Arms "Sling-Shot" with no mention of if they're usable with the Giantslayer.  Based on descriptions in both I'd say they are, so I'm going to change them all to "Slingshot."